### PR TITLE
Fixes #505: Added `Onionscan` analyzer using tor_analyzers docker container

### LINKED ISSUE #505: [New analyzer] - onionscan
GitHub: https://github.com/s-rah/onionscan

```bash
onionscan --jsonReport notarealhiddenservice.onion
```

There are 2 ways to integrate this:
a) as a docker based analyzer
b) compile and store as binary which can executed with subprocess

I'd prefer option `b`, because docker based analyzer should be made only as the last resort.

### PR DESCRIPTION
**Pull request by CypherpunkSamurai**
# Description

`Onionscan` module and configs were added along with `linux-amd64` binary located in `/configurations/bundled/onionscan`. Usage docs were updated and tests were patched.

**Todo:**
- [x] Update `legal_notice.md`
- [x] Provide raw json output sample.
    * [protonmail_output.json.txt](https://github.com/intelowlproject/IntelOwl/files/7921530/protonmail_output.json.txt)

- [ ] ~Update Advanced Usage docs for `verbose` and `torProxyAddress` params.~ (not required)

## Related issues
#505 

## Type of change
- [X] New feature (non-breaking change which adds functionality).

# Checklist

- [X] I have read and understood the rules about [how to Contribute](https://intelowl.readthedocs.io/en/latest/Contribute.html) to this project
- [X] The pull request is for the branch `develop`
- [X] A new analyzer or connector was added, in which case:
    - [X] [Usage](https://github.com/intelowlproject/IntelOwl/blob/master/docs/source/Usage.md) file was updated.
    - [x] [Advanced-Usage](./Advanced-Usage.md) was updated (in case the analyzer/connector provides additional optional configuration).
    - [X] If the analyzer/connector requires mocked testing, `_monkeypatch()` was used in it's class to apply the necessary decorators.
- [x] If external libraries/packages with restrictive licenses were used, they were added in the [Legal Notice](https://github.com/certego/IntelOwl/blob/master/.github/legal_notice.md) section.
- [X] The tests gave 0 errors.
- [X] Linters (`Black`, `Flake`, `Isort`) gave 0 errors. If you have correctly installed [pre-commit](https://intelowl.readthedocs.io/en/latest/Contribute.html#how-to-start-setup-project-and-development-instance), it does these checks and adjustments on your behalf.
- [ ] The commits were squashed into a single one (optional, they will be squashed anyway by the maintainer)
  
### Important Rules
- If your changes decrease the overall tests coverage (you will know after the Codecov CI job is done), you should add the required tests to fix the problem
- Everytime you make changes to the PR and you think the work is done, you should explicitly ask for a review

# Real World Example

- [X] **TODO**

IntelOwl_onionscan1
IntelOwl_onionscan2

### CODE REVIEW

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        self._verbose = params.get("verbose", True)
+        self._tor_proxy_address = params.get("torProxyAddress", None)
+
+    def run(self):
+        """
+        Run Onionscan against target onion domain
+        """
+        # Check for onionscan binary in path/pwd.
+        if which("onionscan"):
+            self.onionscan_binary = "onionscan"
```
Comment (eshaan7) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:40]: is this really required ? we are not installing onionscan so this will never be true.

File: requirements.txt [hunk 2]
```diff
@@ -30,7 +30,7 @@ peepdf-fork==0.4.3
 pdfid==1.0.4
 pefile-fork==2020.12.16
 psycopg2-binary==2.9.1
-pycti==5.1.2a0
+pycti==5.1.3
```
Comment (eshaan7) [inline requirements.txt:33]: pls revert these 2 changes.

File: docker/Dockerfile [hunk 3]
```diff
@@ -40,9 +40,14 @@ RUN touch ${LOG_PATH}/django/api_app.log ${LOG_PATH}/django/api_app_errors.log \
...
     && rm -rf /root/.local
 
+
+# install bundled tools
+COPY configuration/bundled /opt/deploy/bundled
+
 # download yara rules and quark engine rules
 RUN api_app/analyzers_manager/repo_downloader.sh
 
+
```
Comment (eshaan7) [inline docker/Dockerfile:50]: redundant empty line.

File: api_app/analyzers_manager/observable_analyzers/onionscan.py (hunk 1 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:40]: Yeah, that's true. As we are using a docker container and scripts I think its better to remove it.

File: requirements.txt (hunk 2 above)
Comment (CypherpunkSamurai) [reply requirements.txt:33]: Ok. I was actually trying to undo those changes in my commit (that's why the force push). I added that because it wasn't building on my machine.

Will be removed sir :) 👍🏼

File: api_app/analyzers_manager/repo_downloader.sh [hunk 4]
```diff
@@ -7,6 +7,9 @@
 # Example: you may want to add a new repository. Add the clone here
 # Example: you may want to remove some of the rules available in the downloaded repositories. Remove them here.
 
+# Install pre-bundled Onionscan (linux-amd64)
+chmod 755 /opt/deploy/bundled/onionscan
```
Comment (mlodic) [inline api_app/analyzers_manager/repo_downloader.sh:11]: why this line is in this file?

File: docker/Dockerfile [hunk 5]
```diff
@@ -40,6 +41,10 @@ RUN touch ${LOG_PATH}/django/api_app.log ${LOG_PATH}/django/api_app_errors.log \
     # this is cause stringstifer creates this directory during the build and cause celery to crash
     && rm -rf /root/.local
 
+
+# install bundled tools
+COPY configuration/bundled /opt/deploy/bundled
```
Comment (mlodic) [inline docker/Dockerfile:46]: is this really necessary? this file is copied anyway beforehand

File: docker/entrypoint_uwsgi.sh [hunk 6]
```diff
@@ -20,6 +20,9 @@ echo "DEBUG: " $DEBUG
 echo "DJANGO_TEST_SERVER: " $DJANGO_TEST_SERVER
 echo "------------------------------"
 
+# start required daemons
+tor --DataDirectory $(mktemp -d) &
```
Comment (mlodic) [inline docker/entrypoint_uwsgi.sh:24]: honestly I do not like this to happen by default.

Tor should be started only if the user wants to use the OnionScan analyzer. Most of the time the user won't. 

This is also the reason why, in my opinion, this analyzer cannot be enabled by default. We should force the user to choose to activate it if he wants, and let him understand what he is doing. Open a port to Tor cannot just be taken so lightly.

File: docs/source/Usage.md [hunk 7]
```diff
@@ -204,6 +204,7 @@ The following is the list of the available analyzers you can run out-of-the-box.
 * `AbuseIPDB`: check if an ip was reported on [AbuseIPDB](https://www.abuseipdb.com/)
 * `Fortiguard`: scan an observable with the [Fortiguard URL Analyzer](https://www.fortiguard.com/webfilter)
 * `TorProject`: check if an IP is a Tor Exit Node
+* `OnionScan`: check a Tor domain for misconfigurations and info leaks.
```
Comment (mlodic) [inline docs/source/Usage.md:207]: For the reasons mentioned beforehand, I would like to have OnionScan as an analyzer that is optional.

I think one good way to do that is to transform it as a docker analyzer (see https://intelowl.readthedocs.io/en/latest/Advanced-Usage.html#optional-analyzers). Tor and the onionscan binary should be integrated there. They are not necessary in the default image.

However, that would mean to add another optional container. That is not a good thing because, if a user wants to enable all the analyzers, they would already need to run too many containers.

So I think this is the reason why we should integrate OnionScan requirements in an already existing container. We just need to choose which one.
@eshaan7 thoughts?

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 8]
```diff
@@ -0,0 +1,107 @@
...
+        check if tor bootstrap is complete
+        """
+        try:
+            url = "https://check.torproject.org/api/ip"
+            tor_proxy = {
+                "http": "socks5h://127.0.0.1:9050",
+                "https": "socks5h://127.0.0.1:9050",
+            }
+            r = requests.get(url, proxies=tor_proxy)
+            IsTor = json.loads(r.text)["IsTor"]
```
Comment (mlodic) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:98]: better call varaible names without capital letters

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 9]
```diff
@@ -0,0 +1,107 @@
...
+            return IsTor
```
Comment (mlodic) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:99]: just a single `return` for this function please...for better readibility

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+        except Exception as e:
+            logger.info("onionscan:check_tor error: \n%s", str(e))
```
Comment (mlodic) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:101]: use the most recent f-format, not the old %s please

**Review by mlodic — CHANGES_REQUESTED** [review]

File: api_app/analyzers_manager/repo_downloader.sh (hunk 4 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/repo_downloader.sh:11]: Sir, I added that to make sure the bundled tools have correct permissions. Should I move it to the Dockerfile instead?

File: docker/entrypoint_uwsgi.sh (hunk 6 above)
Comment (CypherpunkSamurai) [reply docker/entrypoint_uwsgi.sh:24]: Right, open tor ports can serve as dangerous. Should we use the daemon from a subprocess in onionscan instead?

File: docs/source/Usage.md (hunk 7 above)
Comment (CypherpunkSamurai) [reply docs/source/Usage.md:207]: I think you're right. Having a separate container for all tor enabled tools along with the daemon would help keep the main app separate from the noise that happens to pass through tor.

Adding the daemon to a separate container would be an organized way to handle things, but hard on the CPU resources. Is it suitable if we make installer scripts for separate tools inside the container, and just pass them as command line args during install?

File: api_app/analyzers_manager/observable_analyzers/onionscan.py (hunk 10 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:101]: Ok, Sir.

**Comment by CypherpunkSamurai** [comment]
I'm pushing changes, for now, to keep track of progress (also because I'm using an online vscode instance :P). The onionscan module happens to run ok but takes a lot of time, sometimes timeouts celery.

I agree with Mr. Matteo's suggestions about dockerizing tor enabled tools, so let's discuss it first :)

File: api_app/analyzers_manager/repo_downloader.sh (hunk 4 above)
Comment (mlodic) [reply api_app/analyzers_manager/repo_downloader.sh:11]: yes, I think that line is unrelated to the that .sh file

File: docs/source/Usage.md (hunk 7 above)
Comment (eshaan7) [reply docs/source/Usage.md:207]: > Having a separate container for all tor enabled tools along with the daemon would help keep the main app separate from the noise that happens to pass through tor.

Yep, a new service in `integrations/` for TOR enabled tools makes sense. Similar to how we have one for static analyzers, other for APK analyzers.

Comment (eshaan7) [reply docs/source/Usage.md:207]: I didn't know that onionscan requires an active tor daemon running otherwise I'd have suggested this from the very beginning. That's a mistake on my part for not properly reading through their README. :man_facepalming:

Comment (mlodic) [reply docs/source/Usage.md:207]: As I mentioned, we have already a solution for this cases and I would not like to add other "tricks" to solve this issue.

The solution is: Docker-based analyzers (all the ones that are in the `integration` folder, the ones mentioned in the previous link I shared). 

There is also documentation about how to integrate them: https://intelowl.readthedocs.io/en/latest/Contribute.html#integrating-a-docker-based-analyzer

The concept is that you are separating "problematic" dependencies from the base image and add them in optional images. This is very useful because analyzers should not influence the base image because they are not mandatory to the overall application: most of the users would like to use just some of them and not everyone (they are more than 100).

So, as said, we already have some of this optional images and maybe they are already too many to be run together.
So, instead of creating a new docker-based integration, we could just add OnionScan dependencies and service in an already existing one (like `box-js`). The point is that this case someway is not related to the others. I mean, there are no other analyzers that needs Tor right now. So it would make sense to create a new docker-based analyzer for Tor-related services. If we do that (this is easier, you can do that), then we can try to reduce the amount of the overall number of docker-based analyzers by grouping some ot them together (`box-js` and `apk_analyzers` for instance)

Comment (eshaan7) [reply docs/source/Usage.md:207]: So for this new `tor_analyzers`, you'd need to do 4 things:

- Create new directory inside `integrations/` named `tor_analyzers`. (Refer to `apk_analyzers` directory, you need the same files and same directory structure). This also means onionscan binary should now be kept inside `integrations/tor_analyzers/` as it will be executed from this new container using the flask-shell2http package.
- Convert the `Onionscan` class to inherit from `DockerBasedAnalyzer` class. (Refer to other docker-based observable analyzers such as thug_url.py, etc.)
- Add entry for the `tor_analyzers` integration in the `start.py`.

Comment (mlodic) [reply docs/source/Usage.md:207]: created an issue for this too: https://github.com/intelowlproject/IntelOwl/issues/842

Comment (eshaan7) [reply docs/source/Usage.md:207]: @CypherpunkSamurai so we discussed within ourselves and the final decision is to have a new container for tor enabled services (we will combine/compress other analyzers together for optimization)

So for this new `tor_analyzers`, you'd need to do 4 things:

- Create new directory inside `integrations/` named `tor_analyzers`. (Refer to `apk_analyzers` directory, you need the same files and same directory structure). This also means onionscan binary should now be kept inside `integrations/tor_analyzers/` as it will be executed from this new container using the flask-shell2http package.
- Modify the `Onionscan` class to inherit from `DockerBasedAnalyzer` class. (Refer to other docker-based observable analyzers such as thug_url.py, etc.)
- Add entry for the `tor_analyzers` integration in the `start.py` script.
- Edit `Advanced-Usage.md` add new row in the table under "Optional Analyzers".

Comment (CypherpunkSamurai) [reply docs/source/Usage.md:207]: Ok Sir, On It! 😄

**Comment by CypherpunkSamurai** [comment]
Ok, As the discussions seem to give a green flag to a new category of analyzers I'm overwriting current progress with a fresh new pull from `develop` to clean things up :)

**Comment by CypherpunkSamurai** [comment]
Hello Mr. @mlodic and Mr. @eshaan7, can you please suggest to me how to patch `DockerBasedAnalyzers`?

I have tried the following but these ones aren't working as mock points
* `DockerBasedAnalyzer._docker_run`
* `DockerBasedAnalyzer.__enter__.return_value._docker_run`
* `DockerBasedAnalyzer.return_value._docker_run`
* `self._docker_run`

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 11]
```diff
@@ -0,0 +1,49 @@
...
+        self.args.extend(["-jsonReport", self.observable_name])
+
+        req_data = {
+            "args": self.args,
+        }
+
+        return self._docker_run(req_data=req_data, req_files=None)
+
+    @classmethod
+    def _monkeypatch(cls, *_) -> None:
```
Comment (eshaan7) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:42]: Don't require monkeypatch in DockerBasedAnalyzer.

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 12]
```diff
@@ -0,0 +1,49 @@
...
+    def _onionscan_args_builder(params):
+        verbose = params.get("verbose", True)
+        tor_proxy_address = params.get("torProxyAddress", None)
+        # make request arguments
+        args = []
+        if verbose:
+            args.append("-verbose")
+        if tor_proxy_address:
+            args.extend(["-torProxyAddress", tor_proxy_address])
+        return args
```
Comment (eshaan7) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:29]: No need for this function. Lines 21-22 should be part of `set_params` and line 23-28 part of `run`.

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 13]
```diff
@@ -0,0 +1,49 @@
...
+from tests.mock_utils import patch
+
+
+class Onionscan(ObservableAnalyzer, DockerBasedAnalyzer):
+    name: str = "Onionscan"
+    url: str = "http://tor_analyzers:4004/onionscan"
+    # http request polling max number of tries
+    max_tries: int = 1000
+    # interval between http request polling (in seconds)
+    poll_distance: int = 30
```
Comment (eshaan7) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:14]: In your experience with using onionscan, did u notice it taking a lot of time to return result ? I ask because you've made max_tries 1000 and poll_distance 30 so that's 30,000 seconds or 8 hours :smiling_face_with_tear:

Comment (eshaan7) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:14]: I'd ask you to run 2-3 scans and notice the average time it takes onionscan to return result.

Comment (eshaan7) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:14]: Ideally, the values should be max_tries = 30, poll_distance = 10. So that is 300 seconds or 5 minutes. Which means the `soft_time_limit` should be made ~330.

File: api_app/analyzers_manager/observable_analyzers/onionscan.py (hunk 11 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:42]: Ok Sir

File: api_app/analyzers_manager/observable_analyzers/onionscan.py (hunk 12 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:29]: Ok Sir.

File: api_app/analyzers_manager/observable_analyzers/onionscan.py (hunk 13 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:14]: > In your experience with using onionscan, did u notice it taking a lot of time to return result ? I ask because you've made max_tries 1000 and poll_distance 30 so that's 30,000 seconds or 8 hours smiling_face_with_tear

Yikes, I added that for testing, seems I forgot to remove it.

> I'd ask you to run 2-3 scans and notice the average time it takes onionscan to return result.

> Ideally, the values should be max_tries = 30, poll_distance = 10. So that is 300 seconds or 5 minutes. Which means the `soft_time_limit` should be made ~330.

I have noticed it takes more than 10 minutes, I'll take an average and try if it works out.

Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:14]: I just checked It took more than an hour, and celery just timed it out.

I'm leaving 3 consecutive ones running on a ssh server for a day and I'll return the results tomorrow.

Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:14]: So, it looks like onionscan has a problem with v3 addresses, I checked out its issues and someone had a patch for it (onionscan#162) I added the patch and refactored the source, and it looks like its working. I'm now running tests on the ssh shell and will take an average for the timeouts :)

Comment (eshaan7) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py]: > I checked out its issues and someone had a patch for it (onionscan#162) I added the patch and refactored the source, and it looks like its working.

Really appreciate you going the extra mile with the effort.

So I guess we can make the soft_time_limit and the max_tries x poll_distance to be about 15 minutes.

Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py]: Sure Sir 👍

**Comment by CypherpunkSamurai** [comment]
Sir(s), the DeepSource check seems to be linked to the `develop` branch. Please review my PR for any issues :)

File: integrations/tor_analyzers/app.py [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+app.config.update(CONFIG)
+
+executor = Executor(app)
+shell2http = Shell2HTTP(app, executor)
+
+# binary paths
+onionscan_binary = "./bundled/onionscan"
+
+# with this, we can make http calls to the endpoint: /onionscan
+shell2http.register_command(endpoint="onionscan", command_name=onionscan_binary)
```
Comment (eshaan7) [inline integrations/tor_analyzers/app.py:48]: nitpick: directly assign `command_name="./bundled/onionscan"`.

**Comment by eshaan7** [comment]
Hey, good work! Just one thing that you did not attach a screenshot of the GUI showing that it works.

**Comment by CypherpunkSamurai** [comment]
> Hey, good work! Just one thing that you did not attach a screenshot of the GUI showing that it works.

Oh. My Apologies sir, I got a little sick and was taking time off. I promise I'll attach a screenshot within a day or 2. :P

**Comment by eshaan7** [comment]
> Oh. My Apologies sir, I got a little sick and was taking time off. I promise I'll attach a screenshot within a day or 2. :P

That's not a problem at all. Take your time and be safe!

File: integrations/tor_analyzers/Dockerfile [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+ENV PROJECT_PATH /opt/deploy
+ENV LOG_PATH /var/log/intel_owl/tor_analyzers
+ENV USER tor-user
+
+# Add a new low-privileged user
+RUN useradd -r -s /sbin/nologin ${USER}
+
+# update and install packages
+RUN DEBIAN_FRONTEND=noninteractive apt-get update -qq \
+    && apt-get install --no-install-recommends -y git build-essential gcc pandoc curl tor \
```
Comment (eshaan7) [inline integrations/tor_analyzers/Dockerfile:12]: I see we are installing `tor` here which makes sense but where/when is the tor daemon being launched ?

Comment (CypherpunkSamurai) [reply integrations/tor_analyzers/Dockerfile:12]: We launch the tor daemon from the entry point script, before the python api starts.

File: integrations/tor_analyzers/app.py (hunk 14 above)
Comment (CypherpunkSamurai) [reply integrations/tor_analyzers/app.py:48]: Ok Sir.

**Comment by CypherpunkSamurai** [comment]
Hello Mr @eshaan7, I noticed onionscan tests we taking too long and celery timed out a few times. Turns out onionscan has bugs that don't return proper exit codes (onionscan keeps running after errors/scan completed). I have patched the bug and updated the binary.

I will be uploading screenshots soon.

Sir, I've noticed that having `verbose` set to true by default will cause the json output to get messed up, are you sure we should enable verbose by default?

File: integrations/tor_analyzers/Dockerfile (hunk 15 above)
Comment (eshaan7) [reply integrations/tor_analyzers/Dockerfile:12]: Oh! I missed that file. Thanks for the clarification.

**Comment by CypherpunkSamurai** [comment]
Hello Mr @mlodic and Mr @CypherpunkSamurai. This PR is already completed, Please review this PR :D

File: api_app/analyzers_manager/observable_analyzers/onionscan.py [hunk 16]
```diff
@@ -0,0 +1,35 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+from api_app.analyzers_manager.classes import DockerBasedAnalyzer, ObservableAnalyzer
+
+
+class Onionscan(ObservableAnalyzer, DockerBasedAnalyzer):
+    name: str = "Onionscan"
+    url: str = "http://tor_analyzers:4004/onionscan"
```
Comment (mlodic) [inline api_app/analyzers_manager/observable_analyzers/onionscan.py:9]: 4004 is already used by APK_analyzers. Not a technical problem because it works but for clarity I would prefer this to have a different port from the other docker integrations. `4001` is free.

**Review by mlodic — CHANGES_REQUESTED** [review]
Another change required is:
* update dependabot.yml

File: api_app/analyzers_manager/observable_analyzers/onionscan.py (hunk 16 above)
Comment (CypherpunkSamurai) [reply api_app/analyzers_manager/observable_analyzers/onionscan.py:9]: Sure Sir!

**Comment by CypherpunkSamurai** [comment]
> Another change required is:
> 
> * update dependabot.yml

Hello Mr @mlodic, Please check.

File: .gitpod.yml [hunk 17]
```diff
@@ -0,0 +1,10 @@
+# This configuration file was automatically generated by Gitpod.
+# Please adjust to your needs (see https://www.gitpod.io/docs/config-gitpod-file)
+# and commit this file to your remote git repository to share the goodness with others.
+
+tasks:
+  - init: pip install -r requirements.txt
+vscode:
+  extensions:
+    - ms-python.python
+
```
Comment (mlodic) [inline .gitpod.yml:10]: can you please explain this addition?

Is this related to my request? Because I just asked to update the dependabot.yml. If you check that file, we added all the integrations directory so we need to add this new one too

Comment (CypherpunkSamurai) [reply .gitpod.yml]: Ok Sir, I've removed it.

Comment (mlodic) [reply .gitpod.yml]: yeah but still not changed the `dependabot.yml` :P

Comment (CypherpunkSamurai) [reply .gitpod.yml]: I've updated it sir. Please check.

**Review by mlodic — APPROVED** [review]
